bedrock/firefox/templates/firefox/features/fast.html

```diff
@@ -0,0 +1,122 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/features/base.html" %}
+
+{# L10n: strings for this page are not considered final and aren't ready for localization. #}
+
+{% block page_title %}{{ _('Firefox is faster and more powerful than ever before') }}{% endblock %}
+{% block page_desc %}{{ _('Firefox lets you do more, faster. Our new, powerful multi-process platform easily handles all your tabs without bogging down your computer.') }}{% endblock %}
+{% block page_image %}{{ static('img/firefox/features/page-image-fast.jpg') }}{% endblock %}
+
+{% block page_css %}
+  {{ super() }}
+  {% stylesheet 'firefox-features-hub-detail' %}
+{% endblock %}
+
+{% block body_id %}firefox-features-fast{% endblock %}
+
+{% block features_header_content %}
+  <h1><span>{{ _('We’ve been working out') }}</span></h1>
+  <p class="tagline">{{ _('Firefox is now faster and leaner than ever.') }}</p>
+{% endblock %}
+
+{% block features_list %}
+<section class="features-list-section">
+  <div class="content">
+    <h2 class="section-title"><span>{{ _('Features') }}</span></h2>
+
+    <ul class="features-list">
+      <li class="features-list-item">
+        <img src="{{ static('img/firefox/features/thumbnails/fast/low-memory.jpg') }}" alt="">
```

Comment:
I don't believe we have images high res enough for these. But eventually I imagine these might all be dynamic anyway? I think we're good for now.